force-app/main/default/classes/JobApplicationTriggerHandler.cls

```diff
@@ -154,4 +166,29 @@ public with sharing class JobApplicationTriggerHandler {
     private static Boolean hasContactLinkedToCompanyRef(Id companyIdFromJobApp, Map<Id, Id> accountIdToContactIdMap) {
         return accountIdToContactIdMap.get(companyIdFromJobApp) != null;
     }
+
+        // Calculate Take Home Pay
+    public static void estimateTakeHomePay(List<JobApplication__c> triggerDotNew) {
```

Comment:
Suggest moving this ontop of all the private methods as its one of the functions called from other classes.